### PR TITLE
Capturing Alias creation errors

### PR DESCRIPTION
**Pull request by carolyncole**
Captured excption in the actor stack
Display errors in the flash messages
Removed preserve title order since our title is singular

refs #1075

### CODE REVIEW

File: app/actors/curation_concerns/actors/generic_work_actor.rb [hunk 1]
```diff
@@ -31,6 +32,13 @@ def apply_save_data_to_curation_concern(attributes)
           end
           super
         end
+
+        def capture_alias_errors
+          yield
+        rescue AliasManagementService::Error => error
+          curation_concern.errors.add(:creator, :invalid, message: error.message)
+          return false
```
Comment (awead) [inline app/actors/curation_concerns/actors/generic_work_actor.rb:40]: what happens here? Does the rest of the actor updates/creates continue, or does the whole thing stop? Is the page redirected back to edit or does it continue on to the user dashboard?

Comment (carolyncole) [reply app/actors/curation_concerns/actors/generic_work_actor.rb:40]: @awead I believe that returning false causes the actor stack to stop.  This has nothing to do with the redirect.  The redirect is handled in controller changes I added.

**Comment by carolyncole** [comment]
@awead here is the logic for determining what to do when the actor returns false: https://github.com/samvera/curation_concerns/blob/master/app/controllers/concerns/curation_concerns/curation_concern_controller.rb#L47
and here
https://github.com/samvera/curation_concerns/blob/master/app/controllers/concerns/curation_concerns/curation_concern_controller.rb#L90

File: app/actors/curation_concerns/actors/generic_work_actor.rb (hunk 1 above)
Comment (awead) [reply app/actors/curation_concerns/actors/generic_work_actor.rb:40]: @cam156 are the controller changes in another PR? I don't see any changes related to a redirect.

Comment (carolyncole) [reply app/actors/curation_concerns/actors/generic_work_actor.rb:40]: I take that back the redirect is handled in the controller see the links I added into the PR.

I just changed the fact that the object errors were displayed in the flash.

**Comment by carolyncole** [comment]
Before the PR
screen shot 2017-11-14 at 2 42 44 pm

**Comment by awead** [comment]
@cam156 looks good! It also looks like it the files have been successfully attached so the files requirement is shown as complete, and the user can just fix the metadata problem. Is the files requirement just always green during edit, or is the form actually checking to see if the work has files?

**Comment by carolyncole** [comment]
@awead I have actually found that the file does not seem to still be attached.   That was just my browser not running the javascript.   

Here is an accurate after the PR:
screen shot 2017-11-15 at 7 00 03 am

**Comment by awead** [comment]
@cam156 ok, so it looks like if there's a creator error, files won't get attached. If that seems right to you, I'm 👍

**Comment by carolyncole** [comment]
@awead it is certainly more than I was trying to accomplish with this ticket.  I was not trying to change the overall actor stack flow, just making sure our error was handled.

**Pull request merged**